### DISCUSSION TITLE
Reminder, Notifications & Calendar [Apr 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5544] [Discord Threads]
> 🙋‍♂️ **Hioko**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Reminders & Calendar are not supported. Notification- we have webhook based for activities on the grid (insert / update / delete)   
#### https://docs.nocodb.com/developer-resources/webhooks  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1087078932907560970)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
